crt_portal/cts_forms/templates/forms/complaint_view/side_nav_options.html

```diff
@@ -31,5 +31,12 @@
         </a>
       </li>
       {% endif %}
+      {% if ENABLED_FEATURES.notification_management %}
+      <li class="usa-nav__primary-item" data-ga-event-name="notifications nav item">
+        <a class="crt-landing--menu_link reporting-header {% if page == 'notifications' %} selected {% endif %}" href="/form/notifications">
+          Notifications
```

Comment:
Should this have a "new" tag next to it too?

Reply:
I think I am good with not including this, because we never find a reason take them out haha